.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ui/inspector"]
+	path = ui/inspector
+	url = https://github.com/freya-fs/freya.ui.inspector.git
```

Comment:
I find it strange to have a submodule dependency on a project that depends on other projects in this repo. What uses this dependency? Also, this is very likely the reason the build is broken.